databasetest/mysqltest.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package databasetest provides an ephemeral database server for testing database integration. It's
+// designed to be used in code that needs to work inside and outside Google.
+package databasetest
```

Comment:
This would still need to be in the `mysqltest` package and folder (or have a stub there) to maintain compatibility, if we care.

Reply:
> Dave Revell:
> 
> Hopefully we can keep it. If we change it, we need to change their google3 and github code to use the new name. The name is used in in google3 in some places that will be quite difficult to change, like https://source.corp.google.com/piper///depot/google3/third_party/golang/github_com/abcxyz/pkg/v/v0/mysqltest/BUILD?q=mysqltest%20f:third_party%20f:golang&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental
> 
>